projects/plugins/jetpack/modules/sso/jetpack-sso-invitations.js

```diff
@@ -0,0 +1,16 @@
+/* global jetpack_sso_wpcom_invitations */
+jQuery( document ).ready( function ( $ ) {
+	// $nonce = $( '#jetpack-sso-invite-nonce' ).val();
+	$( '.sso-disconnected-user' ).click( function ( e ) {
+		const { nonce, ajax_url } = jetpack_sso_wpcom_invitations;
+		var $this = $( this );
+		var user_id = $this.attr( 'data-user-id' );
+		$this.html( 'Inviting' );
+		$this.prop( 'disabled', true );
+		e.preventDefault();
+		var data = { action: 'jetpack_invite_user_to_wpcom', user_id, _ajax_nonce: nonce };
+		$.post( ajax_url, data ).done( function () {
```

Comment:
Not sure if needed in this instance, but WP has [a small ajax abstraction](https://github.com/WordPress/wordpress-develop/blob/6daf85302233bc99823b453691379be3a5eb5f22/src/js/_enqueues/wp/util.js#L47-L154) in [the `wp-util` dependency](https://github.com/WordPress/wordpress-develop/blob/6daf85302233bc99823b453691379be3a5eb5f22/src/wp-includes/script-loader.php#L1047) that includes [setting up the ajax url](https://github.com/WordPress/wordpress-develop/blob/6daf85302233bc99823b453691379be3a5eb5f22/src/wp-includes/script-loader.php#L1052-L1053). [Here is an example](https://github.com/WordPress/wordpress-develop/blob/6daf85302233bc99823b453691379be3a5eb5f22/src/js/media/controllers/site-icon-cropper.js#L40-L45) of it in use.

Reply:
I ripped all the ajax logic and went with classic forms, since I'm refreshing anyways.